IM150/admin/admin_forum_tour.php

```diff
@@ -163,7 +165,7 @@ function reorder_pages()
 			$message = prepare_message(trim($message), 1, 1, 1, $bbcode_uid);
 
 			$sql = "UPDATE " . FORUM_TOUR_TABLE . "
-				SET page_subject = '$subject', page_text = '$message', page_access = $page_access
+				SET page_subject = '$subject', page_text = '$message', page_access = '$page_access'

```

Comment:
The access was throwing me for a loop. I'm not real sure what it amounts to. I am in the process of digging as we speak. I never dug this deep in the code in the past so it is all new to me. All I ever did was templates and install mods and hope for the best. This is all learning to me.